stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss

```diff
@@ -69,19 +69,18 @@
         text-transform: uppercase;
 
         &.sub-external {
-          align-items: center;
           justify-content: space-between;
-
+          align-items: center;
           &::after {
-            background: url(#{$img-path-Iconfourback}/external-default.svg) no-repeat 0 bottom/$spacing-32 $spacing-32;
-            content: '';
+            background: url(#{$img-path-Iconfourback}/external-default.svg) no-repeat 0 bottom/1.25rem 1.125rem;
             cursor: pointer;
-            height: $spacing-32;
-            width: $spacing-32;
+            height: 1.125rem;
+            width: 1.25rem;
+            content: "";
           }
-          &:hover{
-            &::after{
-              background: url(#{$img-path-Iconfourback}/external-hover.svg) no-repeat 0 bottom/$spacing-32 $spacing-32;
+          &:hover {
+            &::after {
+              background: url(#{$img-path-Iconfourback}/external-hover.svg) no-repeat 0 bottom/1.25rem 1.125rem;
```

Comment:
@Ashish-srij can we create the placeholder for similar code and use it here

Reply:
no-repeat 0 bottom/1.25rem 1.125rem;
@surbhi90 Only this part is common on above lines. Do you want me to make placeholder for this only.

Reply:
@Ashish-srij Above code in 'subexternal' class like width, height, align-items looks similar to me so we can create the placeholder for that code

Reply:
@surbhi90 Done.